namui-cli/src/services/runtime_project/wasm.rs

```diff
@@ -1,9 +1,11 @@
+use std::path::PathBuf;
+
```

Comment:
Here is empty line

Reply:
I found that rust analyzer automatically make a new line.
https://rust-analyzer.github.io/manual.html#auto-import

I will try coding without this option in other branch and if it works well then I will make PR later about option.